src/features/events/components/EventActionButtons.tsx

```diff
@@ -28,34 +28,35 @@ const EventActionButtons: React.FunctionComponent<EventActionButtonsProps> = ({
   const orgId = event.organization.id;
   const { showConfirmDialog } = useContext(ZUIConfirmDialogContext);
   const router = useRouter();
-
-  const model = useModel((env) => new EventDataModel(env, orgId, event.id));
+  const { cancelEvent, deleteEvent, restoreEvent, setPublished } =
+    useEventMutations(orgId);
```

Comment:
Passing the event ID here means it would not be necessary on lines 39, 43, 46, 50 or 59.